docs/guides/endgame.md

```diff
@@ -0,0 +1,87 @@
+# Endgame
+
+## About
+
+What is the endgame?
+
+Who runs the endgame?
+
+Meet the team:
+
+- Leader (design or development)
+- Sidekicks
+  - 1 from design
+  - 1 from development
+  
+### Rotation
+
+## Releases
+
+### Hotfixes
+
+Quickfix needed now
+
+### Patch releases
+
+During a project cycle, our goal is to group up fixes contributed to the project into weekly patch release cycles. If no new fixes are contributed in a given week, then no new patch would need to be released.
+
+All patches are based on the previous stable git tag. You can find the latest git tag by running the following command in your terminal:
+
+```bash
+git tag -l --sort=-v:refname
+```
+
+The first tag you see at the top of the screen will be the latest tag. In addition, you can use the GitHub UI to view the [latest release](https://github.com/carbon-design-system/carbon/releases/latest). The git tag associated with the latest release will be the one you'll want to use.
+
+You can checkout git tags using `git` by running the following command:
+
+```bash
+git checkout <name-of-tag>
+```
+
+For example:
+
+```bash
+git checkout v10.4.0
+```
+
+You should then create a branch off of this tag that will include that changes for the next patch release. You should name this branch using the following pattern: `chore/release-vX.Y.Z`. The version should be one patch bump higher than the last stable git tag. For example:
+
+- `v10.4.0` becomes `v10.4.1`
+- `v10.4.1` becomes `v10.4.2`
+
+If the last stable git tag was `v10.4.0`, then the branch would be named:
+
+```bash
+git checkout chore/release-v10.4.1
+```
+
+After creating this branch, your goal will be use the [`cherry-pick`](https://git-scm.com/docs/git-cherry-pick) feature from `git` to bring over commits with the following types:
+
+- build
+- ci
+- chore
+- docs
+- fix
+- refactor
+- revert
+- style
+- test
+
+The most important commit type we do not want to bring over is `feat`, as this would be a [`minor`](#minor-releases) release.
```

Comment:
when a PR is merged are the commits squashed? Just curious because when i work on an issue I've been using a mixture of these commit types within a single PR based on the type of work done for that specific commit not based on the PR objective.  In that case, you wouldn't want to bring in my `chore` commits if i also have `feat` commits within the same PR.... This is more for me to learn if I've been doing my commits incorrectly or not

Reply:
@elizabethsjudd it will depend, if folks are intentional about commits then rebasing is fair. If they are using it to track changes that rollup into a single type, then squash is appropriate. The default is definitely to squash and merge though 👍 

You bring up a great point, if the commits are mixed (feat and fix) then rebased would be preferred otherwise the rollup would lose the meaning of the commits, especially if going under a broader feat as it wouldn't get rolled into the cherry picked fixes.